deno.json

```diff
@@ -36,7 +36,7 @@
     "@std/json": "jsr:@std/json@^1.0.0-rc.1",
     "@std/jsonc": "jsr:@std/jsonc@^1.0.0-rc.1",
     "@std/log": "jsr:@std/log@^0.224.3",
-    "@std/media-types": "jsr:@std/media-types@^1.0.0-rc.1",
```

Comment:
Notice: This was previously incorrect. I've raised this issue in #5194 to ensure this doesn't happen again.

Reply:
Can you create another PR for fixing this?